spec/controllers/admin/rebate_forms_controller_spec.rb

```diff
@@ -67,7 +63,7 @@
         end
       end
 
-      context 'pdf' do
+      pending 'pdf' do
```

Comment:
:elephant:  this could probably just be removed rather than pending based on conversations that this functionality isn't part of what we are building any more